classes/WP_CLI/ActionScheduler_WPCLI.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
```

Comment:
missing namespace

Reply:
Oh, I was checking commands. Will get that namespaced as well.